content/en/blog/news/2021-no-docker.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Don't use docker, use kubernetes"
+date: 2021-05-17
+draft: false
+description: If you are developing for kubernetes then don't use docker locally
+categories: [blog]
+keywords: [Community, 2021]
+slug: "dont-use-docker"
+aliases: []
+author: James Strachan
+---
+ 
+Are you developing software that's intended to run on kubernetes? If so we recommend not to use docker on your laptop.
+
+Docker on Windows/MacOS helps you run a VM that can then run linux containers easily. But why bother?
+
+We highly recommend just use a development kubernetes cluster - build and run your containers there instead then you're already closer to production already. 
+
+## Why use kubernetes instead of docker?
+
+* why test on a completely different VM and container orchestrator than production? its better to test on a similar environment to where you are really going to deploy your code
+* test your kubernetes yaml / helm chart and associated configuration at the same time as you run your containers helps you catch mistakes earlier
+    * it's not just about running the container image; it's about lots of other things too like networking, configuration, secrets, storage/volumes, cloud infrastructure, service mesh, liveness/readiness/startup probes - so why not test all of those things rather than just the image?
```

Comment:
```suggestion
* it's not just about running the container image; it's about lots of other things too like networking, configuration, secrets, storage/volumes, cloud infrastructure, service mesh, liveness/readiness/startup probes - so why not test all of those things rather than just the image?
```

Reply:
I kinda wanted that indent ;) - its a kinda related sub point